### PR TITLE
support for Sumer + workers export

### PR DESCRIPTION
**Pull request by ondratra**
This PR adds export of workers and support for Sumer release node.

Warning: this PR likely breaks tools for releases older than Sumer!

### CODE REVIEW

File: src/export_workers.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+interface WorkingGroup {
+  nextWorkerId: () => Promise<WorkerId>
+  workerById: (id: number) => Promise<Worker>
+}
+
+async function getAllWorkers(workingGroupApi: WorkingGroup): Promise<any[]> {
+  const next = ((await workingGroupApi.nextWorkerId()) as WorkerId).toNumber()
+
+  const workers = []
```
Comment (mnaamani) [inline src/export_workers.ts:39]: When workers leave their role, they are removed from the map. So iteration from 0 to next worker id will produce null workers. Instead use the `.keys()` or `.entries()` to get only the existing workers in the map.

Example usage: https://github.com/Joystream/joystream/blob/c5a8bbaf4e4aa19de435278c22fd3530bacedfc9/storage-node/packages/runtime-api/workers.js#L123

File: src/export_workers.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  for (let id = 0; id < next; id++) {
+    const rawWorker = (await workingGroupApi.workerById(id)) as Worker
+
+    workers.push({
+      member_id: rawWorker.member_id,
```
Comment (mnaamani) [inline src/export_workers.ts:47]: we should also include the `id` (workerId) in the export.

**Review by mnaamani — CHANGES_REQUESTED** [review]

File: src/export_workers.ts (hunk 1 above)
Comment (ondratra) [reply src/export_workers.ts:39]: Done.

File: src/export_workers.ts (hunk 2 above)
Comment (ondratra) [reply src/export_workers.ts:47]: Done

**Pull request merged**